x-pack/plugins/cases/common/api/cases/comment/index.test.ts

```diff
@@ -777,6 +778,29 @@ describe('Comments', () => {
         right: defaultRequest,
       });
     });
+
+    it('error when too many comments are passed', () => {
```

Comment:
nit: You can use the `PathReporter` to verify the error message. If I understood correctly we should verify the error messages in this kind of test, right?